README.md

```diff
@@ -116,13 +116,13 @@ You should be able to access the api in your web browser at
 In a new tab in your terminal (on your laptop):
 
 > Optional: log in to your GitHub account, visit
-[https://github.com/jenofdoom/academy-responsive-web](https://github.com/jenofdoom/academy-responsive-web)
+[https://github.com/catalyst/academy-responsive-web](https://github.com/catalyst/academy-responsive-web)
 and hit the fork button in the top right hand corner. Then in the git clone
 command below, replace 'jenofdoom' with your user name.
```

Comment:
That should be 'catalyst' not 'jenofdoom'

Reply:
nice catch. Thanks.

I've updated it ( with a force push )